xonsh/ptk_shell/completer.py

```diff
@@ -99,6 +101,35 @@ def get_completions(self, document, complete_event):
                 disp = comp[pre:].strip("'\"")
                 yield Completion(comp, -plen, display=disp)
 
+    def __find_begidx(self, line, endidx):
+        if len(line[0:endidx]) == 0:
+            return 0
+
+        wordsNonPosix = []
```

Comment:
Python style guidance here is to use `snake_case` for variable names, not `camelCase`.
Additionally, since this variable is populated by the output of `split`, you don't need to define it here

Reply:
removed and renamed